toponline.php

```diff
@@ -1,23 +1,20 @@
 <?php
 require_once 'engine/init.php';
 include 'layout/overall/header.php'; 
-
 if (!$config['toponline']['enabled']) {
 echo 'This page has been disabled at config.php.';
 include 'layout/overall/footer.php';
 	exit();
 }
 $limit = $config['toponline']['limit'];
-$type = $_GET['type'];
-
+$type = isset($_GET['type']);
```

Comment:
You just broke type string comparisons down at line 35+, because isset returns true or false if its set.
This suggestion checks if type is set, if true return the string, else return false. 
This coding method is known as a "conditional value declaration".
```php
$variable = (condition) ? true : false;
```
A mix of `$variable = value` with an if statement built into it.
```suggestion
$type = (isset($_GET['type'])) ? getValue($_GET['type']) : false;
```